Application/Models/Channel.cs

```diff
@@ -1,12 +1,16 @@
+using System.ComponentModel;
 using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
 
 namespace Application.Models;
 
 public class Channel : Chat
 {
+    [DefaultValue("5f95a3c3d0ddad0017ea9291")]
```

Comment:
Может это должно быть имя канала?
```suggestion
    [DefaultValue("Test Channel")]
```

Reply:
блін, як воно сюди взагалі потрапило, я ж робив це в іншій гілці
так, я переплутав